plugin/path_static_account_secrets.go

```diff
@@ -0,0 +1,110 @@
+package gcpsecrets
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/vault/sdk/framework"
+	"github.com/hashicorp/vault/sdk/logical"
+)
+
+func pathStaticAccountSecretServiceAccountKey(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern:    fmt.Sprintf("%s/%s/key", staticAccountPathPrefix, framework.GenericNameRegex("static_account")),
+		Deprecated: true,
+		Fields: map[string]*framework.FieldSchema{
+			"static_account": {
+				Type:        framework.TypeString,
+				Description: "Required. Name of the static_account.",
```

Comment:
I suggest to stick with a common description for the field. The description is `Name to refer to this static account in Vault` above and `Name of the static_account` here. This also applies to `pathStaticAccountSecretAccessToken`.

Reply:
Removed the snake case.